managing_providers/cloud_providers/_topics/enabling_ibm_power_vc_events.md

```diff
@@ -0,0 +1,28 @@
+## Enabling Events in IBM PowerVC
+
+
+To get the IBM PowerVC Events:
+
+1.  Login to the PowerVC CLI and check if the following services are enabled:
+    ```
+    powervc-config image audit
+    powervc-config compute audit
+    powervc-config storage audit
+    ```
+
+2.  If the services are not enabled, use the following commands to enable them:
+    ```
+    powervc-config image audit --enable --restart
+    powervc-config compute audit --enable --restart
+    powervc-config storage audit --enable --restart
+    ```
+
+3.  After enabling the services, you can perform operations for VMs, storage, images, etc.
```

Comment:
I'm a little confused by the "you can perform operations for VMs, storage, images, etc." part of this. If it's not part of the process to enable the services let's remove it or drop it from the enumeration:
```suggestion
After enabling the services {{ site.data.product.title_short }} will be able to collect and display event data.
```